File: JavaProjects/POO_2506_Clase04_HerenciaMultiple/src/Mamifero.java
public class Mamifero {
    private String nombre;
    // Constructor
    public Mamifero(String nombre){
        this.nombre = nombre;
    }
    // Método
    public void Amamantar(){
        System.out.println("Tomo leche proveniente de mi madre.");
    }

    // Getter y Setter
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: C++Projects/POO_2506_Clase02_HerenciaDeClasesEnArchivos/Mamifero.cpp
//
// Created by Diego Cruz on 19/03/2024.
//

#include "Mamifero.h"

// Constructor de la clase
Mamifero::Mamifero(){
    this->nombre = "Por definir";
}

// Setter y Getter
string Mamifero::getNombre(){
    return nombre;
}
void Mamifero::setNombre(string nombre){
    this->nombre = nombre;
}

void Mamifero::setExpresion(string expresion){
    this->expresion = expresion;
}

string Mamifero::getExpresion(){
    return expresion;
}
